samples/template/sample.yaml

```diff
@@ -1,55 +1,58 @@
 sample:
-  name: Sidewalk application template
+  name: Sidewalk template sample
   description: Sample implementing Amazon Sidewalk End Device
 tests:
-  sample.sidewalk.template.lora:
+  sample.sidewalk.template:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    extra_args: CONFIG_APP_PERSISTENT_LINK_MASK=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     tags: Sidewalk
 
-  sample.sidewalk.template.lora.cli:
+  sample.sidewalk.template_update:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-    extra_args: CONFIG_SIDEWALK_CLI=y
+    extra_args: CONFIG_APP_PERSISTENT_LINK_MASK=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
```

Comment:
remove integration_platforms here.
This key is useful when twister is called with -i to check if sample builds. 
This sample is used for testing of DFU, to have the image but with different SHA. it's just copy of regular template

Reply:
Done